src/liballoc/tests/btree/map.rs

```diff
@@ -607,6 +607,267 @@ fn test_range_mut() {
     }
 }
 
+mod test_drain_filter {
+    use super::*;
+
+    const HEIGHT_1_CAPACITY: usize = 11; // node::CAPACITY
+    const HEIGHT_2_MIN_INSERTS: usize = 11 + 1; // just bigger than node::CAPACITY
+    const HEIGHT_3_MIN_INSERTS: usize = 11 + (12 * 11) + 1;
```

Comment:
These constants look like they could/should also be used by other tests in this file?

Reply:
They look poorly named to me, now that I know height has a different meaning (one less) throughout the API. But yes, I think constants like that should be used or that height were accessible to test cases (and to plain users as a measure of complexity, but then definitely with a more intuitive definition).